docs/content/content-blocks.md

```diff
@@ -110,6 +77,69 @@ My content
 ````
 `````
 
+### New Style Admonitions
+
+The admonition syntax above utilises the general [directives syntax](content:myst/directives).
+This is has the advantage of making it consistent with every other directive.
+However, a big disadvantage is that, when working in any standard Markdown editor (or the Jupyter Notebook interfaces),
+the text they contain will not render nicely as standard Markdown.
```

Comment:
```suggestion
the text they contain will not render nicely as standard Markdown (for local Markdown previews).
```

Reply:
I'm not sure if "local" is quite the right word here. As I mention above, it also makes the pages a lot nicer to view on GitHub, or basically anywhere where you are directly viewing the source Markdown file

Reply:
Roger that -- perhaps "(for Markdown previews)" to emphasis this is for preview mechanisms rather than build output.